config/initializers/secure_headers.rb

```diff
@@ -7,7 +7,14 @@
   config.x_permitted_cross_domain_policies = 'none'
 
   # On the development environment only we allow 'https://*' to hop over to identity-pki
-  form_action = Rails.env.development? ? ["'self'", 'https://*'] : ["'self'"]
+  form_action = if Rails.env.development?
+                  ["'self'", 'https://*']
+                else
+                  ["'self'",
+                   'https://*.pivcac.int.identitysandbox.gov',
+                   'https://*.pivcac.staging.login.gov',
+                   'https://*.pivcac.prod.login.gov']
```

Comment:
It is not possible to use a wildcard for the env? E.g. `https://*.pivcac.*.login.gov`?

Reply:
Cause we wouldn't want to leave out the `dev`, `qa`, or any of the devops team's sample envs`.